packages/lesswrong/components/posts/PostsPage/RSVPs.tsx

```diff
@@ -16,7 +16,7 @@ import mapValues from "lodash/mapValues";
 
 const styles = (theme: ThemeType): JssStyles => ({
   body: {
-    marginBottom: 12
+    marginBottom: 48
```

Comment:
used by the EA forum? (maybe not, idk)

Reply:
They said it was okay:
https://lworg.slack.com/archives/CJY3ZKP62/p1722451196764929?thread_ts=1722450904.329019&cid=CJY3ZKP62